6.2/6.2/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _6._2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string StartWord = "информатика";
+            Console.WriteLine(StartWord.Remove(0, 2).Remove(1, 8)
+                + StartWord.Remove(1, 10)
+                + StartWord.Remove(0, 4).Remove(3, 4));
+
+
+            Console.WriteLine(StartWord.Remove(0, 9).Remove(1, 1)
+                + StartWord.Remove(0, 3).Remove(3, 4));
```

Comment:
Локальные переменные методов принято называть с маленькой буквы
```suggestion
            string startWord = "информатика";
            Console.WriteLine(startWord.Remove(0, 2).Remove(1, 8)
                + startWord.Remove(1, 10)
                + startWord.Remove(0, 4).Remove(3, 4));


            Console.WriteLine(startWord.Remove(0, 9).Remove(1, 1)
                + startWord.Remove(0, 3).Remove(3, 4));
```